$ termux-chroot $ cd lm-sensors $ make Segmentation fault $

```
$ termux-chroot
$ cd lm-sensors
$ make
Segmentation fault
$
```

To reproduce this segmentation fault, run the following:
```
termux-chroot
git clone https://github.com/groeck/lm-sensors
cd lm-sensors
make
```
Termux-packages arch:
aarch64
Android version:
6.0.1

![screenshot_20170824-231039](https://user-images.githubusercontent.com/27742457/29698197-c7df0a92-8921-11e7-89f4-600ccefc7bcd.png)

@michalbednarski 
```
$ uname -a
Linux localhost 3.10.84-perf-gcdba782-00725-g9a1b90d #1 SMP PREEMPT Wed Aug 2 13:48:17 CST 2017 aarch64 Android
$ PROOT_NO_SECCOMP=1 termux-chroot
$ cd lm-sensors/
$ make
Segmentation fault
$ exit
logout
$ proot make
proot info: vpid 1: terminated with signal 11
```
https://sdrausty.github.io/logs/proot-log.txt

However, simple with no chroot produces:
`make: *** No rule to make target 'langinfo.h', needed by 'prog/sensors/main.rd'.  Stop.`
![screenshot_20170825-234407](https://user-images.githubusercontent.com/27742457/29738391-5e60636c-89ef-11e7-901a-77ac0213907e.png)

Looks like some assertion failed after first child process completed (after finished `wait4` for `uname -m` make connects to system log (`logcat`), `make` during my test on emulator here didn't use system log (haven't anything to log)).

In order to continue debugging I'd need you to

1. Try with following `Makefile` (put it in new empty directory)
```
MACHINE := $(shell uname -m)
```
(If this file was enough to reproduce segfault continue further steps using this directory,
If it says "`make: *** No targets.  Stop.`" return to `lm-sensors` `Makefile`)

2. Run `make` under `strace` under `proot` and send me result (whenever it works or not)
``` 
proot strace -vs50000 -fo proot-make-trace.txt make
```

3. In case of `make` working correctly in 2. also send system logs (If problem reproduced in 2. traces will already contain these logs)
```
logcat -c
make
logcat -d > proot-make-logcat.txt
```
(`logcat -c` clears log, `logcat -d` prints existing logs and exits without waiting for new ones to come)

@michalbednarski Thank you for your time and guidance.
```
$ mkdir new
$ cd new
$ vi Makefile
$ make
make: *** No targets.  Stop.
$
```
```
$ cd lm-sensors/
$ proot strace -vs50000 -fo proot-make-trace.txt make
proot info: vpid 1: terminated with signal 11
$ 
```
Output at https://sdrausty.github.io/logs/proot-make-trace.txt 🙂

@michalbednarski Thank you for the update. 
```

$ unzip ../proot_5.1.106-2_aarch64.zip
Archive:  ../proot_5.1.106-2_aarch64.zip
  inflating: proot_5.1.106-2_aarch64.deb  
$ dpkg --install proot_5.1.106-2_aarch64.deb
(Reading database ... 91366 files and directories currently installed.)
Preparing to unpack proot_5.1.106-2_aarch64.deb ...
Unpacking proot (5.1.106-2) over (5.1.106-2) ...
Setting up proot (5.1.106-2) ...
Processing triggers for man (1.14.3) ...
$ cd lm-sensors/
$ make
/data/data/com.termux/files/usr/bin/sh: 1: /sbin/ldconfig: Permission denied
make: *** No rule to make target 'langinfo.h', needed by 'prog/sensors/main.rd'.  Stop.
$ proot strace -vs50000 -fo proot-make-trace.txt make
/data/data/com.termux/files/usr/bin/sh: 1: /sbin/ldconfig: not found
make: *** No rule to make target 'langinfo.h', needed by 'prog/sensors/main.rd'.  Stop.
$ termux-chroot
$ cd lm-sensors/
$ make
/data/data/com.termux/files/usr/bin/sh: 1: /sbin/ldconfig: not found
make: *** No rule to make target 'langinfo.h', needed by 'prog/sensors/main.rd'.  Stop.
$
```
Looks like this issue is resolved. Thank you 🙂

--- a/packages/proot/build.sh
+++ b/packages/proot/build.sh
@@ -1,11 +1,11 @@
 TERMUX_PKG_HOMEPAGE=http://proot.me/
 TERMUX_PKG_DESCRIPTION="Emulate chroot, bind mount and binfmt_misc for non-root users"
 # Just bump commit and version when needed:
-_COMMIT=ccf3d7ab4804c1d01dd4d8f970da033a872c3152
+_COMMIT=6671bfed4ddcbd393d8ca7b2754d58e41ed9595b
 TERMUX_PKG_VERSION=5.1.106
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL=https://github.com/termux/proot/archive/${_COMMIT}.zip
-TERMUX_PKG_SHA256=16c2a450dae0c321c2b949bab322ebcd8bff4d78ce989ef3024f2235d2b7fbbf
+TERMUX_PKG_SHA256=e3565bd6789659d86106d6e08eb204d8606af94664aebcb20dcf68a1852c4ba2
 TERMUX_PKG_FOLDERNAME=proot-$_COMMIT
 TERMUX_PKG_DEPENDS="libtalloc"
 

